index.js

```diff
@@ -10,7 +10,8 @@ var noop = function() {}
 module.exports = function(db) {
   var feed = {}
   var lock = mutexify()
- 
+  var valueEncoding = db.options.valueEncoding || 'binary'
```

Comment:
yea. db.options.valueEncoding is just a default value. utf-8, json, binary, anEncoderObject is always accepted